.github/dependencies.yaml

```diff
@@ -5,8 +5,9 @@
 "1.9":
   K8S_VERSION: "1.29"
   JUJU_VERSION: "3.6"
-  UATS_BRANCH: "main"
+  UATS_BRANCH: "track/1.9"
 latest:
   K8S_VERSION: "1.31"
   JUJU_VERSION: "3.6"
+  URL: "https://raw.githubusercontent.com/canonical/bundle-kubeflow/refs/heads/main/releases/latest/edge/bundle.yaml"
```

Comment:
Do we need this? This has a default https://github.com/canonical/charmed-kubeflow-uats/pull/158/files#diff-5c7ae307b96dc381b02c53b1d0ff17d914554ba74fe6a736cb36a87efd341ad3R6 (even for latest).

Reply:
Not strictly speaking, but I would kinda use this to make sure the changes work as expected

Reply:
ACK, we could still remove it if we run tests once without issues (meaning it's using the URL passed). Still, I do not have a strong preference.